src/controllers/admin/webhook.ts

```diff
@@ -5,10 +5,24 @@ import { StatusCodes } from 'http-status-codes';
 import { EventTracker, eventTracker } from '../../services/track/tracker.js';
 import type { INotifyMessage } from '../../types/track.js';
 import { OperationNameEnum } from '../../types/constants.js';
-import { buildSubmitOperation } from '../../services/track/helpers.js';
+import { SubscriptionService } from '../../services/admin/subscription.js';
+import { CustomerService } from '../../services/api/customer.js';
+import { LogToHelper } from '../../middleware/auth/logto-helper.js';
+import { UserService } from '../../services/api/user.js';
+import type { SupportedPlanTypes } from '../../types/admin.js';
+import type { CustomerEntity } from '../../database/entities/customer.entity.js';
+import { buildSubscriptionData } from '../../services/track/helpers.js';
 
 dotenv.config();
 export class WebhookController {
+	private readonly stripe: Stripe;
+
+	constructor() {
+		this.stripe = new Stripe(process.env.STRIPE_SECRET_KEY);
+	}
+
+	public static instance = new WebhookController();
+
 	public async handleWebhook(request: Request, response: Response) {
```

Comment:
We don't need to initialise the Stripe client here since it's already initialised in two other places.
1. HTTP Handlers - `response.locals.stripe`
2. Constructor of this -> `this.stripe`